### ISSUE TITLE
HTMLElement.rootElement is deprecated in Atom 1.19.0-beta0

### ISSUE THREAD

**Issue by pingshunhuangalex** [justmoon/relative-numbers#39]
The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `rootElement` and access
the editor contents directly instead.
```
HTMLElement.rootElement (C:\Users\Pingshun Huang\AppData\Local\atom\app-1.19.0-beta0\resources\app\src\text-editor-element.js:23:10)
LineNumberView._updateSync (C:\Users\Pingshun Huang\.atom\packages\relative-numbers\lib\line-number-view.coffee:101:50)
_updateSync (C:\Users\Pingshun Huang\.atom\packages\relative-numbers\lib\line-number-view.coffee:3:59)
<unknown> (C:\Users\Pingshun Huang\.atom\packages\relative-numbers\lib\line-number-view.coffee:83:26)
ViewRegistry.performDocumentUpdate (<embedded>:20022:15)
<unknown> (<embedded>:19838:65)
```

**Comment by avindra** [comment]
Still an issue in beta 3

**Comment by justmoon** [comment]
Duplicate of #39 - fixed in 0.8.1.
